js_modules/dagster-ui/packages/ui-core/src/instance/InstanceConcurrency.tsx

```diff
@@ -112,6 +135,79 @@ type DialogAction =
     }
   | undefined;
 
+const RunConcurrencyContent = ({
+  hasRunQueue,
+  runQueueConfig,
+}: {
+  hasRunQueue: boolean;
+  runQueueConfig: any;
+}) => {
+  if (!hasRunQueue) {
+    return (
+      <>
+        <Box padding={{vertical: 16, horizontal: 24}} border="bottom">
+          <Subheading>Run concurrency</Subheading>
+        </Box>
+        <Box>
+          Run concurrency is not supported with this run coordinator. To enable run concurrency
+          limits, configure your instance to use the <Mono>QueuedRunCoordinator</Mono> in your{' '}
+          <Mono>dagster.yaml</Mono>. See the{' '}
+          <a href="https://docs.dagster.io/deployment/dagster-instance#queuedruncoordinator">
+            QueuedRunCoordinator documentation
+          </a>{' '}
+          for more information.
+        </Box>
+      </>
+    );
+  }
+
+  const info_content = (
+    <Box padding={{vertical: 16, horizontal: 24}}>
+      Run concurrency can be set in your run queue settings. See the{' '}
+      <a href="https://docs.dagster.io/guides/limiting-concurrency-in-data-pipelines#configuring-run-level-concurrency">
+        run concurrency documentation
+      </a>{' '}
+      for more information.
+    </Box>
+  );
+
+  const settings_content = runQueueConfig ? (
+    <MetadataTableWIP>
+      <tbody>
+        <tr>
+          <td>Max concurrent runs:</td>
+          <td>
+            <Mono>{runQueueConfig.maxConcurrentRuns}</Mono>
+          </td>
+        </tr>
+        <tr>
+          <td>Tag concurrency limits:</td>
+          <td>
+            {runQueueConfig.tagConcurrencyLimitsYaml ? (
+              <StyledRawCodeMirror
+                value={runQueueConfig.tagConcurrencyLimitsYaml}
+                options={{readOnly: true, lineNumbers: true, mode: 'yaml'}}
+              />
+            ) : (
+              '-'
+            )}
+          </td>
```

Comment:
Interesting -- I'm curious to see what this looks like in the UI with the inline CodeMirror, can you provide a screenshot?